views/layout.jade

```diff
@@ -213,14 +213,11 @@ html
             li.pull-left
               a#youtube-icon(href="https://www.youtube.com/channel/#{settings.youtube}", target='_blank')
                 span.glyphicon.fa.fa-youtube
-     .col-md-4
+      .col-md-4
```

Comment:
https://github.com/iquidus/explorer/issues/235

it can fix this cosmetic issue. i agree on this commit.

Reply:
LGTM